### DISCUSSION TITLE
issues with runpod notebook

### DISCUSSION THREAD

**Discussion by vesperialuna** [TheLastBen/fast-stable-diffusion#2339] [General]
hi ben,

i have been using your notebook in runpod quite a while now and it's great. recently i stumbled upon this issue and not sure how to go about it. whether using the restored backup from huggingface or when i start a fresh sd repository, the error is almost the same. something like this:
![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/121345493/2ce82a22-884f-46ea-9506-ef9c1256c606)

this happens when i am trying to generate an image. never got to see the image.

any idea why it happened? how do i go about it?

thanks in advance.

**Comment by TheLastBen** [comment]
does the issue persist if you try a different pod ?

**Reply by vesperialuna** [reply]
i tried on 3 different pods so far (the quadro rtx5000, a4000, and a4500. all giving me the same error. tried both new rep and restored ones.

**Reply by TheLastBen** [reply]
I couldn't reproduce the error, make sure you're not installing any dependency before running inference

**Reply by vesperialuna** [reply]
i think i know the issue. if i disable the 'Always save all generated images' i will get that error
![image](https://github.com/TheLastBen/fast-stable-diffusion/assets/121345493/eb0cb3f6-8a96-4845-8c9b-358c13e1416a)
is it a bug?

**Reply by TheLastBen** [reply]
there is a temporary fix, in a new cell, run `!mkdir /temp/gradio`

**Reply by vesperialuna** [reply]
should i set the cell after installing/restoring the repo?

**Reply by TheLastBen** [reply]
yes
